internal/controller/vrg_kubeobjects.go

```diff
@@ -301,6 +301,7 @@ func (v *VRGInstance) executeCaptureSteps(result *ctrl.Result, pathName, capture
 	requestsProcessedCount := 0
 	requestsCompletedCount := 0
 	labels := util.OwnerLabels(v.instance)
+	labels[util.VeleroKubevirtMetadataOnlyBackupLabel] = "true"
```

Comment:
This is will be used only for kubevirt, and ignored for other workloads?

Reply:
Yes, it is a kubevirt group label and only interpreted by the velero kubevirt plugin. 

I need the knob only to allow the checks in the kubevirt velero plugin if required (no use case today). It is just a good idea to have a switch instead of rebuilding ramen to try that.